deps/src/test/java/com/microsoft/azure/sdk/iot/deps/serializer/ParserUtilityTest.java

```diff
@@ -679,23 +680,20 @@ public void mapToJsonElementEmptyMapSucceed() throws ClassNotFoundException
     public void getSimpleDateStringFromDateGivenNullDateThrows()
     {
         //act
-        ParserUtility.getDateStringFromDate(null);
+        ParserUtility.getUTCDateStringFromDate(null);
     }
 
-    //Tests_SRS_PARSER_UTILITY_34_043: [The provided date shall be converted into this format: "yyyy-MM-dd'T'HH:mm:ss".]
     @Test
-    public void getSimpleDateStringFromDateSuccess()
+    public void getSimpleDateStringFromDateWithTimezone()
```

Comment:
Now this unit test does string -> date -> string in order to verify that the string we get from the service is preserved